vinyldns/client_test.go

```diff
@@ -29,4 +31,26 @@ func TestNewClientFromEnv(t *testing.T) {
 	if client.SecretKey != "secretKey" {
 		t.Error("NewClientFromEnv should set a SecretKey from the environment")
 	}
+	if client.Host != "https://vinyldns-api.com" {
```

Comment:
the only thing I could _possibly_ think of is that this could be something like `SchemeHost` instead of `Host` so people wouldn't think something like `vinyldns-api.com` is valid but that's probably too clunky for the trouble

Reply:
@atecce Thanks - I agree that `Host` is a bit poorly named, though I _think_ that's beyond the scope of this PR given that it's been a `go-vinyldns` `Client` field from the beginning. I agree though; ideally it would be called something like `APIURL` or `APIBase` or something more accurate.